src/oatpp/network/tcp/client/ConnectionProvider.cpp

```diff
@@ -123,7 +123,7 @@ provider::ResourceHandle<data::stream::IOStream> ConnectionProvider::get() {
 
     if(clientHandle >= 0) {
 
-      if(connect(clientHandle, currResult->ai_addr, static_cast<int>(currResult->ai_addrlen)) == 0) {
+      if(connect(clientHandle, currResult->ai_addr, static_cast<socklen_t>(currResult->ai_addrlen)) == 0) {
```

Comment:
Let's use a newly introduced `v_sock_size`

Reply:
merged master that has this